Dockerfile

```diff
@@ -39,16 +39,11 @@ LABEL name="Elastic Cloud on Kubernetes" \
       description="Elastic Cloud on Kubernetes automates the deployment, provisioning, management, and orchestration of Elasticsearch, Kibana, APM Server, Beats, and Enterprise Search on Kubernetes" \
       io.k8s.description="Elastic Cloud on Kubernetes automates the deployment, provisioning, management, and orchestration of Elasticsearch, Kibana, APM Server, Beats, and Enterprise Search on Kubernetes"
```

Comment:
Sorry I missed this initially. These label choices where motivated by the need to override existing labels in the UBI base image. I think now that we are moving back to distroless there is no need for these often redundant (`description` and `io.k8s.description` and `summary`, `io.k8s.display-name` and `name`) labels. 

I wonder if we should use OCI standardised labels on this docker image instead (for the UBI one we have to stick with the current set of labels to keep the overrides)
  https://github.com/opencontainers/image-spec/blob/main/annotations.md

Reply:
This is a good point.  I've removed the overrides, and added OCI annotations as described in link, and groups the k8s labels together alphabetically.  lmk if I've covered them all.

Also verified that the golang:1.x image we base things from does not have any preexisting labels:
```
            "Labels": {}
```